source/persistency/hdf5_functions.h

```diff
@@ -96,13 +96,21 @@ typedef struct{
     float     final_z;
   } step_info_t;
 
+  typedef struct{
+    int32_t event_id;
+    unsigned int sensor_id;
+    unsigned int time_bin;
+    unsigned int charge;
```

Comment:
Why these three are `unsigned`? Can any of these be negative?

Reply:
But `event_id` won't be negative either, right?

Reply:
No, but it may be larger than an unsigned int, therefore we used `int32` in the simulation. In fact now it is even larger, but in PETALO we don't do very large runs.